Makefile

```diff
@@ -9,14 +9,21 @@ DEBUG ?= false
 HUB_IMAGE ?= nspccdev/neofs
 HUB_TAG ?= "$(shell echo ${VERSION} | sed 's/^v//')"
 
+GO_VERSION ?= 1.16
+
 BIN = bin
-DIRS= $(BIN)
+DIRS = $(BIN)
+
+RELEASE = release
+ARCHS = amd64
 
 # List of binaries to build.
 CMDS = $(notdir $(basename $(wildcard cmd/*)))
 BINS = $(addprefix $(BIN)/, $(CMDS))
+RELEASE_BINS = $(foreach ARCH,$(ARCHS),$(addsuffix -$(ARCH), $(addprefix $(RELEASE)/, $(CMDS))))
```

Comment:
Why it's a separate list? Shuldn't the regular `bin` and release `bin` be the same?

Reply:
because i think of `make all` as "I want to have neofs binaries in my local env for using it/testing it", so it shouldn't be with arch suffixes and I don't want to have all arch variations to be build

on the other hand, `make prepare-release` is something that shouldn't be used by devs every day, so I think it should be used separately